src/all/mangaplus/src/eu/kanade/tachiyomi/extension/all/mangaplus/MangaPlus.kt

```diff
@@ -412,11 +421,11 @@ class MangaPlus(
         json.decodeFromString(body.string())
     }
 
-    private val SharedPreferences.imageQuality: String
-        get() = getString("${QUALITY_PREF_KEY}_$lang", QUALITY_PREF_DEFAULT_VALUE)!!
+    private fun SharedPreferences.imageQuality(): String = getString("${QUALITY_PREF_KEY}_$lang", QUALITY_PREF_DEFAULT_VALUE)!!
 
-    private val SharedPreferences.splitImages: Boolean
-        get() = getBoolean("${SPLIT_PREF_KEY}_$lang", SPLIT_PREF_DEFAULT_VALUE)
+    private fun SharedPreferences.splitImages(): Boolean = getBoolean("${SPLIT_PREF_KEY}_$lang", SPLIT_PREF_DEFAULT_VALUE)
```

Comment:
having custom property getter vs just a regular function, are effectively the same thing